tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Whitespace;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+use PhpCsFixer\WhitespacesFixerConfig;
+
+/**
+ * @author Vladimir Boliev <voff.web@gmail.com>
+ *
+ * @internal
+ * @coversNothing
```

Comment:
Should be `@covers \PhpCsFixer\Fixer\Whitespace\MethodChainingIndentationFixer`.